### ISSUE TITLE
Implementation of Version Tracking

### ISSUE THREAD

**Issue by WillBriggs** [SanAndreasNoLimits/SANLRPG#54]
Implementation of Version Tracking. At the moment, we are just adding resources willy-nilly. We need some sort of version tracking system implementing, or even just a version number that we update when a major feature/feature set is added.

**Comment by 1B0Y** [comment]
Agreed. The last thing we need is to be pushing untested code out to the server, which may introduce some kind of exploit for an unfair advantage. I would suggest creating a development server for us to work on, along with another branch which the dev server will pull from, and work from that. Once a feature is fully developed and confirmed as non-buggy by a tester, then it can be pushed towards live either singularly or in a "patch" of other updates.

**Comment by CallumDawson** [comment]
How could this be achieved without developers having to update a fourth thing (aside from the code, GitHub issues, and the GitHub changelog)?
